### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by RaghavAwasthi**
Added Do's and Don'ts to the Student Guide

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,4 +49,4 @@ We ask you to send your pull requests (PRs) to the forks there. This way, we can
 ## GCI admins
 - Ali Ok (aliok AT redhat DOT com; Gitter username: aliok)
 - Anuj Garg (anujgargcse AT gmail DOT com; Gitter username: KeenWarrior)
-- Georgios Zaronikas-Karagiannis (gzaronik AT redhat DOT com; Gitter username: gzaronikas )
+- Georgios Zaronikas-Karagiannis (gzaronik AT redhat DOT com; Gitter username: TODO )
```
Comment (KeenWarrior) [inline README.md]: Please revert this gitter username change part

File: README.md [hunk 2]
```diff
@@ -27,6 +27,19 @@ Quick note, it is considered a good practice to use Google before contacting men
...
 
+## Don’t do these things
+- Don't use foul language to any student or mentor in any case. Always be polite to other community members and mentors.
+- You should never argue with mentors with regarding your work . They are highly experienced people if they ask you to do some change   you should follow their advice.
+- At times it might happen that there could be some delay with mentors reviewing your work. You should always consider the fact that      mentors are also humans they also have life and other work to do.You should *NOT* be constantly texting them to respond.
+- **WARNING**  Never try to copy someone else’s work .
+
+## Do these things
+- When Stuck in a task you should always try to find out the solution to the problem by googling it out before pinging mentors.
+- Try to have the discussions related to a project in the project's chatroom.
```
Comment (aliok) [inline README.md]: We won't have dedicated chat rooms for projects, right @KeenWarrior?

File: README.md [hunk 3]
```diff
@@ -27,6 +27,19 @@ Quick note, it is considered a good practice to use Google before contacting men
 
 For any troubles you're having with the organization, mentors or other students, feel free to contact the GCI admins (details below).
 
+## Don’t do these things
+- Don't use foul language to any student or mentor in any case. Always be polite to other community members and mentors.
+- You should never argue with mentors with regarding your work . They are highly experienced people if they ask you to do some change   you should follow their advice.
```
Comment (aliok) [inline README.md]: I think this line could be written "softer". Sounds too harsh.

**Comment by aliok** [comment]
Thanks @RaghavAwasthi 
I have some questions/comments but overall it is great!

File: README.md (hunk 3 above)
Comment (RaghavAwasthi) [reply README.md]: Okay I will make the changes asap

File: README.md (hunk 1 above)
Comment (RaghavAwasthi) [reply README.md]: Yep

File: README.md (hunk 3 above)
Comment (RaghavAwasthi) [reply README.md]: I have made the changes.

File: README.md (hunk 2 above)
Comment (divyanshkul) [reply README.md]: I think we should make new chat rooms for the projects, it would make it much more organised. CC @KeenWarrior

File: README.md [hunk 4]
```diff
@@ -27,6 +27,19 @@ Quick note, it is considered a good practice to use Google before contacting men
 
 For any troubles you're having with the organization, mentors or other students, feel free to contact the GCI admins (details below).
 
+## Don’t do these things
+- Don't use foul language to any student or mentor in any case. Always be polite to other community members and mentors.
+- Try to avoid arguing with mentors regarding your task submission.
+- At times there might be delay in reviewing a task .You should *NOT* be constantly texting them to respond.
+- **WARNING**  Never try to copy someone else’s work .
```
Comment (divyanshkul) [inline README.md]: @RaghavAwasthi Please make it
-**WARNING**  Never try to copy someone else’s work. Plagiarism is strictly prohibited, you will be banned from the program.

File: README.md (hunk 2 above)
Comment (KeenWarrior) [reply README.md]: @aliok we may later decide to have multiple rooms. But I wouldn't suggest to do that in the initial run.

File: README.md (hunk 4 above)
Comment (RaghavAwasthi) [reply README.md]: +1 👍

File: README.md (hunk 2 above)
Comment (RaghavAwasthi) [reply README.md]: Okay so, in that case, I will remove this line for the time being.

**Review by divyanshkul — APPROVED** [review]
LGTM
Thank you @RaghavAwasthi

**Review by KeenWarrior — APPROVED** [review]

**Pull request merged**